### DISCUSSION TITLE
Suggestion for PHD thesis

### DISCUSSION THREAD

**Discussion by mjuma14** [aub-mind/arabert#168] [General]
Hello, 
Thanks for this community I really appreciate it
Amm, I read a lot about Bert, last think I think to work with text classification of hate arab tweets,
Is it good? Or any idea to contribute in this area, 
Really want to do some useful research not just to pass ..

Thanks

**Comment by yassmine-lam** [comment]
Hi,

I just came across ur question and I wanted to share some thoughts that might be beneficial to u.
I just want to say that there a lot to do for Arabic NLP tasks in general using deep learning and transfer learning models like BERT. So u can absolutely choose hate speech detection in Arabic using BERT or other transfer learning methods as a subject for ur PhD. Start by reading surveys and related work in this field  (on English and other languages datasets not only Arabic). Then try to enhance these methods on the dataset u choose to work with (u should prepare a list of available Arabic hate speech datasets). You can even provide a new dataset if you assume the existing ones are insufficient for further experiments. You can also produce a Review paper in this area...  

Good luck
